samples/BikeSharingApp/README.md

```diff
@@ -1,4 +1,73 @@
-# Bike Sharing Sample: Iteratively Develop and Debug Microservices in Kubernetes
+# Deploy the Bike Sharing sample application to Azure Kubernetes Service
 
-## Quickstart
-[Quickstart: Team development on Kubernetes using Azure Dev Spaces](https://docs.microsoft.com/en-us/azure/dev-spaces/quickstart-team-development).
\ No newline at end of file
+*Bike Sharing* is a microservices-based sample application that helps showcase the sandboxing capabilities of [Azure Dev Spaces](https://aka.ms/devspaces). 
+
+Follow the steps below to deploy this sample app to Azure Kubernete Service (AKS).
+
+## Prerequisites
+* [Azure subscription](https://azure.microsoft.com/free)
+* [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+* [Helm 2.13 (or greater)](https://github.com/helm/helm/blob/master/docs/install.md)
+
+## Configure Azure resources
+
+1. **Create an Azure Kubernetes Service cluster.** You must create an AKS cluster in a [supported region](https://docs.microsoft.com/azure/dev-spaces/about#supported-regions-and-configurations). The below commands create a resource group called `MyResourceGroup` and an AKS cluster called `MyAKS` in the `eastus` region.
+
+    ```bash
+    az login
+    az group create --name MyResourceGroup --location eastus
+    az aks create -g MyResourceGroup -n MyAKS --location eastus --disable-rbac --generate-ssh-keys
+    ```
+
+1. **Enable Azure Dev Spaces on the AKS cluster.**
+    ```bash
+    az aks use-dev-spaces -g MyResourceGroup -n MyAKS --space master --yes
+    ```
+
+## Deploy the BikeSharing sample app
+
+1. **Create a fork of this GitHub repo.** It's better to create a *fork* for this exercise so that you can save your own commits and repo configuration.
+
+1. **Clone your fork and navigate into its directory**.
+    ```bash
+    git clone https://github.com/USERNAME/dev-spaces
+    cd dev-spaces/samples/BikeSharingApp/
+    ```
+
+1. **Retrieve the HostSuffix for the `master` dev space.**
+    ```bash
+    azds show-context
+
+    Name                ResourceGroup     DevSpace  HostSuffix
+    ------------------  ----------------  --------  -----------------------
+    MyAKS               MyResourceGroup   master    fedcab0987.eus.azds.io
+    ```
+
+1. **Update the Helm chart with your HostSuffix.** Open [`charts/values.yaml`](https://github.com/Azure/dev-spaces/blob/master/samples/BikeSharingApp/charts/values.yaml) and replace all instances of `<REPLACE_ME_WITH_HOST_SUFFIX>` with the HostSuffix value you retrieved earlier. Save your changes and close the file.
```

Comment:
Based on our dicussion this morning, we should tell them to commit at this point.

Reply:
I added this as the last step so that the user can validate they have a successful deployment before committing to master.